proposals/0016-assets/compile-compute-shader-breakdown.md

```diff
@@ -0,0 +1,68 @@
+<!-- {% raw %} -->
+
+# C interface for HLSL compiler as a library - Work item breakdown #1
+## (Adoption by ML team and ability to compile compute shader using library)
+
+* Parent Proposal: [0016](../0016-c-interface-compiler-library.md)
+* Author(s): [Cooper Partin](https://github.com/coopp)
+* Sponsor: [Cooper Partin](https://github.com/coopp)
+* Status: **Under Consideration**
+* Impacted Project(s): (Clang)
+
+## Introduction
+
+An effort is underway to modernize compilation of HLSL based shaders by adding
+HLSL compilation to clang. This document is a sub proposal for 
+[0016](../0016-c-interface-compiler-library.md) to break down the work
+required to achieve the first milestone of adopotion by the ML team and
+ability to compile a compute shader using clang as a library.
+
+The Issues created here will be ordered to help contruct a roadmap that can be
+leveraged towards defining a realistic timeline to completion.
+
+## Issues
+
+Implementation work in this list is assumed to include tests!
```

Comment:
TBH, I think that some of the steps here could likely be combined, but this glosses over the bigger roadmap questions of what the entire DXC API surface covers.